libmozdata/config.py

```diff
@@ -11,12 +11,21 @@
 
 
 class Config(object):
-    def get(section, option, default=None):
-        raise NotImplementedError
+    def __init__(self):
+        self.local_config = {}
+
+    def set_default(self, section, option, value):
+        self.local_config[(section, option)] = value
+
+    def get(self, section, option, default=None):
+        if (section, option) in self.local_config:
+            return self.local_config[(section, option)]
+        return None
```

Comment:
We could utilize the dictionary's `get()` method:

```suggestion
        return self.local_config.get((section, option), default)
```